MonacoEditorComponent/Monaco/Severity.cs

```diff
@@ -1,19 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Monaco
-{
-    /// <summary>
-    /// https://microsoft.github.io/monaco-editor/api/enums/monaco.severity.html
-    /// </summary>
-    public enum Severity
```

Comment:
This was a Monaco change, eh? What's it called now? Would it be good to document it in the breaking changes?

Reply:
Yes. Severity has been renamed to MarkerSeverity.